shoov/modules/custom/shoov_restful/plugins/restful/file/screenshots_upload/1.0/ShoovScreenshotsUploadResource.class.php

```diff
@@ -59,6 +59,20 @@ protected function createScreenshotNode(array $files) {
 
     $wrapper->field_baseline_name->set($request['baseline_name']);
 
+    if (!$vocabulary_id = shoov_repository_get_vocabulary_by_repo('screenshots_tags', $repo_node)) {
+      $vocabulary_id = shoov_repository_create_screenshot_tags_vocabulary($node);
```

Comment:
remove, assume it is there. please open a follow up issue to write script to update existing repos